README.md

```diff
@@ -720,6 +742,39 @@ using a script such as `tests/test_ddebug_archimedes/vfc_dderrors.py`, which
 returns a [quickfix](http://vimdoc.sourceforge.net/htmldoc/quickfix.html)
 compatible output with the union of _ddmin_ instructions.
 
+## Find Optimal precision with vfc_precexp and vfc_report
+
+The ``vfc_precexp`` script tries to minimize the precision for each function call 
+of a code compiled with verificarlo. To use vfc_precexp, you need to compile your code
+with the ``--inst-func`` anf to write two scripts as for the delta-debug:
```

Comment:
anf -> and

Reply:
please add a link to "delta-debug" to point to the relevant section in the README.md

Reply:
it's done